### ISSUE TITLE
Add time to SGF files

### ISSUE THREAD

**Issue by ursi** [playhex/playhex#97]
HexGui has a display for the time of each player, so I assume it can be added somehow.

**Comment by alcalyn** [comment]
Yes, adding time info was not as easy as I thought, so still in todo :p 

What I noted, here is all relevant SGF nodes for time control:

- at game level:
  - TM total time
  - OT overtime
  - LC for byo yomi, number of periods
  - LT for byo yomi, time of a period
example: `TM[259200]OT[86400 fischer]`

- at move level:
  - WL / BL remaining time for black or white after move played
  - OW / OB number of moves still to make in this byo-yomi period for black or white in byo yomi period (when there is like 30s / 5 moves)

https://homepages.cwi.nl/~aeb/go/misc/sgf.html

**Comment by alcalyn** [comment]
I tried to add time info in SGF file for testing, seems to be ignored `TM[259200]OT[86400 fischer]` and `B[i13]BL[500]WL[400]`

``` sgf
(;FF[4]CA[UTF-8]TM[259200]OT[86400 fischer]AP[PlayHex:0.0.0]SO[PlayHex.org]GM[11]SZ[17]PB[Aaaa]PW[Random bot]DT[2025-08-20]HA[0]RE[W+Resign]
;B[i13]BL[500]WL[400];W[swap-pieces]BL[300]WL[200];B[k12];W[o17];B[i12];W[q15];B[i9];W[o4];B[h8];W[q3];B[g7];W[h11];B[pass];W[d4];B[f11];W[o9])
```

**Comment by ursi** [comment]
Oh, it has a spot for it on the UI, so I just assumed it was. Could we at least get the time in the replays on playhex?

**Comment by alcalyn** [comment]
Yes, also the time is not elapsing, but instead going up...

For time in game replay: https://github.com/playhex/playhex/issues/123
